modules/install/pages/sizing-general.adoc

```diff
@@ -215,12 +215,14 @@ Based on the above formula, these are the suggested sizing guidelines:
 This tells you that the RAM requirement for the whole cluster is 7 GB.
 Note that this amount is in addition to the RAM requirements for the operating system and any other software that runs on the cluster nodes.
 
-==== CPU Overhead
+[#cpu-overhead]
+== CPU Overhead
```

Comment:
Just wanted to check that promoting to h2 is deliberate, as it's not mentioned in ticket spec.

Reply:
Yep, that's fine. It brings it in line with the other headings.